pkg/didcomm/protocol/issuecredential/service.go

```diff
@@ -58,11 +58,11 @@ type customError struct{ error }
 // transitionalPayload keeps payload needed for Continue function to proceed with the action
 type transitionalPayload struct {
 	// Protocol instance ID
-	PIID      string
-	StateName string
-	Msg       service.DIDCommMsgMap
-	MyDID     string
-	TheirDID  string
+	PIID      string                `json:"piid"`
```

Comment:
is there really a need for these struct tags?

Reply:
I am reusing the payload. I am saving data as `transitionalPayload` structure. But when I read the data I am using a different `Action` structure. 
https://github.com/hyperledger/aries-framework-go/blob/master/pkg/didcomm/protocol/issuecredential/service.go#L539

Reply:
Removed!